blog/content/post/2024-04-11-aem_phase4.md

```diff
@@ -0,0 +1,257 @@
+---
+title: TrenchBoot Anti Evil Maid - Phase 4
+abstract: "This blog post marks the completion of the next phase of TrenchBoot
+           Anti Evil Maid project for Qubes OS. This time the project focused
+           on AMD platforms, which is something that wasn't possible with the
+           original solution based on tboot."
+cover: /covers/trenchboot-logo.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2024-04-11
```

Comment:
```suggestion
date: 2024-05-17
```

Reply:
https://github.com/3mdeb/news-and-ideas/pull/592/commits/36bcaf6c3b67070087da7359d81493343e171ad8